LibOS/shim/test/fs/manifest.template

```diff
@@ -31,3 +31,7 @@ sgx.trusted_files.libpthread = file:../../../../Runtime/libpthread.so.0
 sgx.trusted_files.libgcc_s = file:/lib/x86_64-linux-gnu/libgcc_s.so.1
 
 sgx.allowed_files.tmp_dir = file:tmp/
+
+sgx.protected_files_key = ffeeddccbbaa99887766554433221100
+sgx.protected_files.input = file:tmp/pf_input
```

Comment:
It would be nice to support multiple protected-directory paths, probably separated by a delimiter(like colon or semi-colon).